contracts/zero-ex/contracts/src/migrations/InitialMigration.sol

```diff
@@ -28,25 +28,42 @@ import "./LibBootstrap.sol";
 
 /// @dev A contract for deploying and configuring a minimal ZeroEx contract.
 contract InitialMigration {
-    /// @dev The deployed ZereEx instance.
-    ZeroEx private _zeroEx;
+
+    /// @dev The allowed caller of `deploy()`. In production, this would be
+    ///      the governor.
+    address public immutable deployer;
+    /// @dev The real address of this contract.
+    address private immutable _implementation;
+
+    /// @dev Instantiate this contract and set the allowed caller of `deploy()`
+    ///      to `deployer_`.
+    /// @param deployer_ The allowed caller of `deploy()`.
+    constructor(address deployer_) public {
```

Comment:
Not a blocker here, but it could be good to rethink our naming conventions. We've had feedback that our use of underscores is confusing. In one of the other repos we've nixed the `_` for variables altogether and prefixed globals with `g`.

Reply:
Yeah I'm down to make this change down the line.